tools/generator-fluid/app/templates/package.json

```diff
@@ -31,11 +31,11 @@
     "@types/puppeteer": "1.3.0",
     "@types/react": "^16.9.15",
     "@types/react-dom": "^16.0.11",
-    "jest": "^26.4.2",
+    "jest": "^25.5.4",
     "jest-junit": "^10.0.0",
     "jest-puppeteer": "^4.3.0",
     "puppeteer": "^1.20.0",
-    "ts-jest": "^26.2.0",
+    "ts-jest": "^25.5.0",
```

Comment:
Curious, why version goes down?

Reply:
ts-jest 26.2 is [not compatible with TypeScript 3.7](https://github.com/kulshekhar/ts-jest/blob/59bf7bc30887c44a5f97c5648b4522bf728a3de5/package.json#L73), thus the downgrade to a compatible version. That required a downgrade of jest too to stay compatible with the ts-jest downgrade.

Reply:
This is now moot after #4930.